### ISSUE TITLE
loose search of a target file

### ISSUE THREAD

**Issue by Flakke123** [khast3x/h8mail#39]
* h8mail version: Version 2.3 - "SEASON_PRIMER"
* Python version: Python 3.7.4
* Operating System: Kali

### Description

I tryd to check a list of nicknames that saved in a textfile (.txt)

### What I Did

```
h8mail -t targets.txt --loose
 but h8mail uses the filename as a target itself

`[>] h8mail is up to date
[>] Targets:
['targets.txt']
[~] Removing duplicates
[~] Target factory started for targets.txt
[~] HIBP free tier will stop working on the 2019/08/18
[~] You can already use a purchased API key using h8mail (config file)
[!] Could not contact HIBP for targets.txt
401
[!] hunter.io (pubic API) error: targets.txt
list index out of range
[!] Could not contact emailrep for targets.txt`
```
but i can´t drop the "--loose" command because the file contains no email adresses

what am i doing wrong?
thanks for help in advance!

**Comment by Flakke123** [comment]
i used the workaround to copy all (something like40) nicknames directly to the commandline

**Comment by khast3x** [comment]
Currently AFK, re-opening this issue to check it out later. 
The `--loose` option is typically used with local search, such as `-lb` or `-gz`.

You can find more detailed examples in this blog post: 
https://khast3x.club/h8mail/2019/05/26/h8mail-release/#loose-search-and-inputs

Cheers!

**Comment by khast3x** [comment]
Fixed in 2.5.3
